style.css

```diff
@@ -90,6 +174,31 @@ input[type=text] {
   display: none;
 }
 
+.contact {
+  font-size: 30px;
+  padding: 18px;
+}
+
+.contact li {
+  list-style: circle;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #818080;
+  padding: 10px;
+  position: fixed;
+  bottom: 0;
+  width: 88%;
+}
+
+.footer-text {
+  margin-bottom: 10px;
+  font-size: 30px;
+}
+
 @media screen and (max-width: 768px) {
   .mobile {
```

Comment:
- [ Optional] I think that your app will look much better if you decrease the size of the footer text.